doc/sphinx/practical-tools/utilities.rst

```diff
@@ -1,482 +1,1054 @@
 beside proof development, tactics writing or documentation.
 
-\section[Using Coq as a library]{Using Coq as a library}
 
-In previous versions, \texttt{coqmktop} was used to build custom
-toplevels --- for example for better debugging or custom static
-linking. Nowadays, the preferred method is to use \texttt{ocamlfind}.
+Using Coq as a library
+----------------------
+
+In previous versions, ``coqmktop`` was used to build custom
+toplevels - for example for better debugging or custom static
+linking. Nowadays, the preferred method is to use ``ocamlfind``.
 
 The most basic custom toplevel is built using:
-\begin{quotation}
-\texttt{\% ocamlfind ocamlopt -thread -rectypes -linkall -linkpkg
-  -package coq.toplevel toplevel/coqtop\_bin.ml -o my\_toplevel.native}
-\end{quotation}
-
-For example, to statically link LTAC, you can just do:
-\begin{quotation}
-\texttt{\% ocamlfind ocamlopt -thread -rectypes -linkall -linkpkg
-  -package coq.toplevel -package coq.ltac toplevel/coqtop\_bin.ml -o my\_toplevel.native}
-\end{quotation}
-and similarly for other plugins.
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-
-\section[Building a \Coq\ project with {\tt coq\_makefile}]
-{Building a \Coq\ project with {\tt coq\_makefile}
-\label{Makefile}
-\ttindex{Makefile}
-\ttindex{coq\_Makefile}
-\ttindex{\_CoqProject}}
-
-The majority of \Coq\ projects are very similar: a collection of {\tt .v}
-files and eventually some {\tt .ml} ones (a \Coq\ plugin).  The main piece
-of metadata needed in order to build the project are the command
-line options to {\tt coqc} (e.g. {\tt -R, -I},
-\SeeAlso Section~\ref{coqoptions}). Collecting the list of files and
-options is the job of the {\tt \_CoqProject} file.
-
-A simple example of a {\tt \_CoqProject} file follows:
-
-\begin{verbatim}
--R theories/ MyCode
-theories/foo.v
-theories/bar.v
--I src/
-src/baz.ml4
-src/bazaux.ml
-src/qux_plugin.mlpack
-\end{verbatim}
-
-Currently, both \CoqIDE{} and Proof General (version $\geq$ 4.3pre) understand
-{\tt \_CoqProject} files and invoke \Coq\ with the desired options.
-
-The {\tt coq\_makefile} utility can be used to set up a build infrastructure
-for the \Coq\ project based on makefiles.  The recommended way of
-invoking {\tt coq\_makefile} is the following one:
-
-\begin{verbatim}
-coq_makefile -f _CoqProject -o CoqMakefile
-\end{verbatim}
+
+::
+
+   % make tools/tolink.cmx
+   % ocamlfind ocamlopt -thread -rectypes -linkall -linkpkg -package coq.toplevel \
+                 -I tools tolink.cmx toplevel/coqtop_bin.ml -o my_toplevel.native
+
+For example, to statically link |L_tac|, you can add it to ``tools/tolink.ml`` and use:
+
+::
+
+   % make tools/tolink.cmx
+   % ocamlfind ocamlopt -thread -rectypes -linkall -linkpkg \
+                 -package coq.toplevel -package coq.ltac    \
+                 -I tools tolink.cmx toplevel/coqtop\_bin.ml -o my\_toplevel.native
+
+We will remove the need for the ``tolink`` file in the future.
```

Comment:
Sorry but it looks once again like what was ported wasn't the most recent version of this section. See the diff.